dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -218,4 +236,286 @@ public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request
         return responseResource.response( "true" );
 	}
 
+    /**
+     * Deletes all bundles by identifier
+     * Note: the response will be notified by socket message
+     * @param request   {@link HttpServletRequest}
+     * @param response  {@link HttpServletResponse}
+     * @param deleteBundlesByIdentifierForm {@link DeleteBundlesByIdentifierForm} contains the set of bundle ids to delete.
+     */
+	@DELETE
+    @Path("/ids")
+    @Produces("application/json")
+    public Response deleteBundlesByIdentifiers(@Context   final HttpServletRequest request,
+                                               @Context   final HttpServletResponse response,
+                                               final DeleteBundlesByIdentifierForm  deleteBundlesByIdentifierForm) {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requiredBackendUser(true)
+                .requiredFrontendUser(false)
+                .requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .init();
+
+        Logger.info(this, "Deleting the bundles: " + deleteBundlesByIdentifierForm.getIdentifiers()
+                + " by the user: " + initData.getUser().getUserId());
+
+        final DotSubmitter dotSubmitter = DotConcurrentFactory
+                .getInstance().getSubmitter(BUNDLE_THREAD_POOL_SUBMITTER_NAME);
+        final Locale locale = LocaleUtil.getLocale(request);
+
+        dotSubmitter.execute(() -> {
+
+            try {
+
+                this.deleteBundleByIdentifier(deleteBundlesByIdentifierForm, initData);
+                this.sendSuccessDeleteBundleMessage(deleteBundlesByIdentifierForm.getIdentifiers().size(), initData, locale);
+            } catch (DotDataException e) {
+
+                Logger.error(this.getClass(),
+                        "Exception on deleteBundlesByIdentifiers, couldn't delete the identifiers: "
+                                + deleteBundlesByIdentifierForm.getIdentifiers() +
+                                ", exception message: " + e.getMessage(), e);
+
+                this.sendErrorDeleteBundleMessage(initData, locale, e);
+            }
+        });
+
+        return Response.ok(new ResponseEntityView(
+                "Removing bundles in a separated process, the result of the operation will be notified")).build();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Removing bundles in a separated process, the result of the operation will be notified" appears 5 times in this file; the first occurrence is on line 284](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)